windows/x_walletconfig_w.cpp

```diff
@@ -437,6 +455,40 @@ bool WalletConfig::applyChanges() {
             return false;   // need to be restarted. Just want to cancell caller of caller changes state operation
         }
 
+        bool need2updateAutoStartMQSEnabled = ( autoStartMQSEnabled != ui->start_mqs->isChecked() );
+        if (need2updateAutoStartMQSEnabled) {
+            state->updateAutoStartMQSEnabled(ui->start_mqs->isChecked());
+        }
+        autoStartMQSEnabled = ui->start_mqs->isChecked();
+
+        bool need2updateAutoStartKeybaseEnabled = ( autoStartKeybaseEnabled != ui->start_keybase->isChecked() );
+        if (need2updateAutoStartKeybaseEnabled) {
+            state->updateAutoStartKeybaseEnabled(ui->start_keybase->isChecked());
+        }
+        autoStartKeybaseEnabled = ui->start_keybase->isChecked();
+
+        if (logoutTimeout != currentLogoutTimeout) {
+            // 1. Update the config...
+            util::ConfigReader reader;
+            QString configFN = config::getMwcGuiWalletConf();
+            if ( !reader.readConfig( configFN ) ) {
+                control::MessageBox::messageText(nullptr, "Internal Error",
+                                             "Unable to update wallet config file " + configFN );
+            }
+
+            bool updateOk = reader.updateConfig("logoutTimeout", QString::number(currentLogoutTimeout));
+
+            if (!updateOk) {
+                control::MessageBox::messageText(nullptr, "Error", "Wallet unable to set the selected logout time." );
+            }
+
```

Comment:
Also you need to change the current value for this running app. 
state::StateMachine     mwc-qt-wallet/state/statemachine.h/cpp   manage timeout,   it is using  config::getLogoutTimeMs() to read the value.
You should update   logoutTimeMs  value at mwc-qt-wallet/core/Config.cpp  to change timeout for the running app.